index.html

```diff
@@ -1,44 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-   <meta charset="UTF-8">
-   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-   <title>Nelson Mandela</title>
-   <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-   <main id="main">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Nelson Mandela</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <!-- Nelson Mandela Project Presents -->
```

Comment:
[OPTIONAL] please remove the comment lines. it is not a good way to leave comment lines in codebase